test/run_test.py

```diff
@@ -1808,6 +1809,17 @@ def __str__(self):
                 print_to_stderr("Emiting td_test_failure_stats")
                 emit_metric("td_test_failure_stats", test_stats)
 
+                # Is this an excessive number of things to emit? probably.  But
+                # it was also really easy to write
+                emit_metric(
```

Comment:
I'm thinking it would be to have this data put inside the previous emit_metrics heuristic

The benefit to merging them is that some fields would be beneficial to queries of either table (e.g. to see what tests were prioritized above a given test). 

Currently joining the two tables is non-trivial and potentially error prone since we don't have the job id, and putting it in one table would make those queries easy.

Reply:
sorry is the ask just to combine the two emit metrics into one?